src/actions/posts.js

```diff
@@ -938,15 +938,18 @@ export function getPostsAfterWithRetry(channelId, postId, page = 0, perPage = Po
 }
 
 // Note that getProfilesAndStatusesForPosts can take either an array of posts or a map of ids to posts
-export async function getProfilesAndStatusesForPosts(posts, dispatch, getState) {
-    if (!posts) {
+export async function getProfilesAndStatusesForPosts(postsArrayOrMap, dispatch, getState) {
+    const posts = Object.values(postsArrayOrMap);
```

Comment:
`postsArrayOrMap` will this be only object?

Reply:
It can be an array of posts or an object mapping post IDs to posts. `Object.values` converts it into an array of posts either way. It's always done that, but it used to do it later in the process.

I did just realize this doesn't null check any more though so I'll fix that